packages/mgt/package.json

```diff
@@ -20,20 +20,19 @@
   "main": "./dist/es6/index.js",
   "types": "./dist/es6/index.d.ts",
   "module": "./dist/es6/index.js",
```

Comment:
Do they still exist? What are they used for?

Reply:
yes they do exist, they're created by tsc, they are used when installing via npm.

the esm bundle from rollup is a single file that the browser can download and pull elements out of.

Reply:
i.e. if you're using npm and importing, the files linked via /dist/es6/index.js are used, if you're in the browser you use mgt.js